src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -225,7 +232,7 @@ private EntityTypeColumn handleSingleSelectCustomField(String name,
                                                          String sourceViewName,
                                                          String sourceViewExtractor,
                                                          List<ValueWithLabel> columnValues) {
-    String filterValueGetter = sourceViewExtractor + " ->> '" + refId + "'";
+    String filterValueGetter = sourceViewExtractor + STRING_EXTRACTOR + refId + "'";
```

Comment:
nit, since you're here:

could we make STRING_EXTRACTOR something like `%s ->> '%s'` and we do `STRING_EXTRACTOR.formatted(sourceViewExtractor, refId)`?